### ISSUE TITLE
[Bug] Remove MD5 usages

### ISSUE THREAD

**Issue by Redth** [dotnet/maui#2518]
Essentials SecureStorage uses MD5 still - we should evaluate and remove usages here

**Comment by yunusefendi52** [comment]
Is this changes going to be in xamarin.essentials as well or should I make a new issue for it?

**Comment by Redth** [comment]
No, we are not backporting these types of changes to legacy Xamarin.Essentials.

**Comment by v-longmin** [comment]
Thank you so much for your bug report! Unfortunately, we cannot reproduce your issue with the information above, could you provide a sample project and screenshots or more details that we can reproduce your issue? That should greatly speed up the process, thanks!

**Comment by rachelkang** [comment]
Implemented in #2807
